src/display/text_layer.js

```diff
@@ -499,9 +465,16 @@ var renderTextLayer = (function renderTextLayerClosure() {
    * @param {boolean} enhanceTextSelection
    * @private
    */
-  function TextLayerRenderTask(textContent, container, viewport, textDivs,
-                               enhanceTextSelection) {
+  function TextLayerRenderTask({ textContent, textContentStream, container,
+                                 viewport, textDivs, textContentItemsStr,
+                                 enhanceTextSelection, }) {
+    this._layoutTextLastFontSize = null;
```

Comment:
Can we please try to order the properties in a more logical way here, by first initializing the ones from the parameters and then the rest after?
```javascript
this._textContent = textContent;
this._textContentStream = textContentStream;
this._container = container;
this._viewport = viewport;
this._textDivs = textDivs || [];
this._textContentItemsStr = textContentItemsStr || [];
this._enhanceTextSelection = !!enhanceTextSelection;

this._reader = null;
// All other existing (and new) properties here...
```